functional-test/src/test/groovy/org/rundeck/tests/functional/api/project/RunCommandSpec.groovy

```diff
@@ -26,11 +26,37 @@ class RunCommandSpec extends BaseContainer{
         def runResponseBody = runResponse.body().string()
         def parsedResponseBody = mapper.readValue(runResponseBody, RunCommand.class)
         def newExecId = parsedResponseBody.execution.id
+        waitForExecutionStatus(newExecId)
```

Comment:
could we use the "Execution" returned from this method to validate the status in line 34 instead of doing another api call?